src/main/kotlin/com/r3/developers/csdetemplate/MyFirstFlow.kt

```diff
@@ -26,25 +25,25 @@ class Message(val sender: MemberX500Name, val message: String)
 // MyFirstFlow should inherit from RPCStartableFlow, which tells Corda it can be started via an RPC call
 class MyFirstFlow: RPCStartableFlow {
 
-    // It is useful to be able to log messages from the flows for debugging.
+    // Log messages from the flows for debugging.
     private companion object {
         val log = contextLogger()
     }
 
     // Corda has a set of injectable services which are injected into the flow at runtime.
     // Flows declare them with @CordaInjectable, then the flows have access to their services.
 
-    // JsonMarshallingService provides a Service for manipulating json
+    // JsonMarshallingService provides a service for manipulating JSON
     @CordaInject
     lateinit var jsonMarshallingService: JsonMarshallingService
 
-    // FlowMessaging provides a service for establishing flow sessions between Virtual Nodes and
-    // sending and receiving payloads between them
+    // FlowMessaging provides a service that establishes flow sessions between virtual nodes which
+    // sends and receives payloads between them.
     @CordaInject
     lateinit var flowMessaging: FlowMessaging
 
-    // MemberLookup provides a service for looking up information about members of the Virtual Network which
-    // this CorDapp is operating in.
+    // MemberLookup provides a service that looks for information about members of the virtual network which
+    // this CorDapp operates in.
```

Comment:
// MemberLookup provides a service for looks up information about members of the virtual network which
    // this CorDapp operates in.
    
    Think of MemberLookup as a "yellow pages" service